### DISCUSSION TITLE
Duda sobre backend

### DISCUSSION THREAD

**Discussion by EduGarcia502** [Launch-X-Latam/MisionFrontEnd#361] [Q&A]
donde sera la trasmicion de backend estoy algo perdido con eso y no se a donde dirigirme, disculpen alguien puede orientarme 
muchas gracias

**Comment by danielcatzinvi** [comment]
Puedes ver la grabación en este enlace o en el apartado de teams "grabaciones"

https://web.microsoftstream.com/channel/128255fd-081d-48b9-8ba9-479008367142
